### PR TITLE
Player number can be set when creating room

### PR DESCRIPTION
**Pull request by edwardycl**

### CODE REVIEW

File: frontend/src/app/game-create.component.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
     <label>Password(optional)</label>
     <input [(ngModel)]="password" name="password" type="password">
   </div>
+  <div>
+    <label>Player number</label>
+    <input [(ngModel)]="player_number" name="player_number" type="number">
```
Comment (tirr-c) [inline frontend/src/app/game-create.component.html]: 드랍다운 박스를 사용한다?

Comment (edwardycl) [reply frontend/src/app/game-create.component.html]: 일단 구조만 정해놓고 구체적 디자인은 mdl branch에서 하려고

**Comment by skystar-p** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit fa0e6d1 has been approved by `skystar-p`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit fa0e6d18a4e789e67ea9fef3bf7563114b359371 with merge 86dcf43aa4534a41ab92a895f8d150420883f3a1...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/310104717?utm_source=github_status&utm_medium=notification)
Approved by: skystar-p
Pushing 86dcf43aa4534a41ab92a895f8d150420883f3a1 to master...

**Pull request merged**